Missing dist files in 29.3 release

<!-- Please make sure the new issue you are opening is not a duplicate, and **remove** this notice! -->

It looks like the `dist` files are missing from releases after 29.2.4.

**Steps to reproduce**

```
npm install @gitbeaker/node@29.3
ls node_modules/@gitbeaker/node
# LICENSE.md   README.md    package.json
```

**Expected behaviour**
```
npm install @gitbeaker/node@29.2
ls node_modules/@gitbeaker/node
# LICENSE.md   README.md    dist         package.json
```

**Actual behaviour**

Since `package.json` points to `dist` files (in `main` etc.), it is not possible to import/require the package.
```
node:internal/modules/cjs/loader:356
      throw err;
      ^

Error: Cannot find module '/path/to/project/node_modules/@gitbeaker/node/dist/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (node:internal/modules/cjs/loader:348:19)
    at Function.Module._findPath (node:internal/modules/cjs/loader:561:18)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:926:27)
    at Function.Module._load (node:internal/modules/cjs/loader:773:27)
    at Module.require (node:internal/modules/cjs/loader:1012:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (/path/to/project/test.js:4:18)
    at Module._compile (node:internal/modules/cjs/loader:1108:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:988:32) {
  code: 'MODULE_NOT_FOUND',
  path: '/path/to/project/node_modules/@gitbeaker/node/package.json',
  requestPath: '@gitbeaker/node'
}
``` 

**Possible fixes**

Maybe something related to #1851.

--- a/.gitlab-ci.yml
+++ b/.gitlab-ci.yml
@@ -154,6 +154,7 @@ canary:
 
 # Release
 release:
+  needs: ['build']
   only:
     refs:
       - master

